dist/tools/pr_check/pr_check.sh

```diff
@@ -25,7 +25,7 @@ else
 fi
 
 SQUASH_COMMITS="$(git log $(git merge-base HEAD "${RIOT_MASTER}")...HEAD --pretty=format:"    %h %s" | \
-                  grep -i -e "^    [0-9a-f]\{7\} .\{0,2\}SQUASH" -e "^    [0-9a-f]\{7\} .\{0,2\}FIX")"
+                  grep -i -e "^    [0-9a-f]\{7,9\} .\{0,2\}SQUASH" -e "^    [0-9a-f]\{7,9\} .\{0,2\}FIX")"
```

Comment:
How about using `\+` instead? If the 9-length short-hashes are also ambiguous (which is why in your case the 7-length wasn't returned) Git will lengthen to the next value.

Reply:
> How about using \+ instead?

Yeah, it's simpler and also works. Will use that if you prefer.

Reply:
Yes, I always prefer the most generic solution if it works as well or better ;-)

Reply:
Done